tags/state/test/test.browser.js

```diff
@@ -0,0 +1,117 @@
+const sinon = require("sinon");
+const { render, cleanup, fireEvent } = require("@marko/testing-library");
+const { expect, use } = require("chai");
+use(require("chai-dom"));
+use(require("sinon-chai"));
+
+const EventEmitter = require("events").EventEmitter;
+const simpleState = require("./fixtures/simple");
+const assignmentState = require("./fixtures/assignment");
+const booleanState = require("./fixtures/boolean");
+const multiState = require("./fixtures/multi");
+const largeFunction = require("./fixtures/largeFunction");
+const noInit = require("./fixtures/noInitializer");
+
+describe("browser", () => {
+  afterEach(cleanup);
+
+  describe("State update", () => {
+    it("Updates simple state", async () => {
+      const { rerender, getByText } = await render(simpleState, {
+        value: 1
+      });
+      const button = getByText("Increment");
+
+      fireEvent.click(button);
+      await rerender();
+      expect(getByText("2").tagName).to.equal("DIV");
```

Comment:
Not a big deal, but `getBy` already does an assertion that the element is there and I don't think the test really cares about the wrapping div.

Could use `queryBy` which does not use an assertion to make the intent of these tests more clear, eg

```
expect(queryByText("2")).to.exist
```